chemreg/substance/models.py

```diff
@@ -70,9 +70,9 @@ class Substance(CommonInfo):
         substance_histories (QuerySet): One to Many Substance history resources (not implemented yet)
     """
 
-    preferred_name_regex = "^[a-zA-Z0-9 =<>\\-':.,^%&/{}[\\]()+?=]{3,}$"
-    display_name_regex = "^[a-zA-Z0-9 =<>\\-':.,^%&/{}[\\]()+?=]{3,}$"
-    casrn_regex = "^[0-9]{2,7}-[0-9]{2}-[0-9]$"
+    preferred_name_regex = r"^[a-zA-Z0-9 =<>\-\*_':.,^%&\/{}[\]]{3,}$"
+    display_name_regex = r"^[a-zA-Z0-9 =<>\-\*_':.,^%&\/{}[\]]{3,}$"
+    casrn_regex = r"^[0-9]{2,7}-[0-9]{2}-[0-9]$"
```

Comment:
Looks like you deleted a chunk of the regex. When I run pytest --create-db the tests fail due to not accepting the ? character.

()+?=

I like the edits to turn these strings into python regex strings and eliminating the need for double backslashes.

Reply:
Thanks, mb, "=" is already captured. Will fix, adding back "()+?"